Automattic/jetpack
# Podcast helper: Make methods non-static

Fixes 363-gh-Automattic/dotcom-manage

#### Changes proposed in this Pull Request:
The `Jetpack_Podcast_Helper` class currently define static methods that cannot be mocked with phpunit, making the unit tests that cover functionality using the helper harder to maintain. See D53816-code as an example.

This PR refactors the class so method are no longer static allowing them to be mocked by unit tests. It also makes all the private methods protected for the same reason. There are no changes in the functionality.

#### Jetpack product discussion
N/A.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
There are no functionality changes in this PR, so make sure there are no regressions:
- In a new post, add a Podcast player
- Add a feed URL such as http://feeds.feedburner.com/lerendezvousjeux
- The block should display nicely in the editor and on the frontend.

#### Proposed changelog entry for your changes:
No.
